.github/workflows/dissect-ci-template-self-hosted.yml

```diff
@@ -109,42 +84,18 @@ jobs:
     steps:
       - if: ${{ inputs.run_tests == true && inputs.deb-packages != '' && ! startsWith(matrix.os, 'windows') }}
```

Comment:
`inputs.run_tests == true` is  checked on line 66.

Where is `deb-packages` coming from? Shouldn´t it be declared in `workflow_call`?